modules/swagger-codegen/src/main/resources/scalatra/logback.xml

```diff
@@ -0,0 +1,13 @@
+<configuration>
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
Suggest also to add rolling file appender, it's more useful.

Reply:
Thanks!

I added.